src/main/java/frc/robot/subsystems/gpm/Arm.java

```diff
@@ -279,12 +270,12 @@ public void periodic() {
         motors[0].setControl(m_request.withOutput(dutyCycle));
 
         // report the arm angle in radians
-        SmartDashboard.putNumber("abs value", encoder.getAbsolutePosition());
+        //SmartDashboard.putNumber("abs value", encoder.getAbsolutePosition());
         
 
         // TODO: Clean these up when not needed.
         // report dutycycle
-        SmartDashboard.putNumber("arm pow", dutyCycle);
+       // SmartDashboard.putNumber("arm pow", dutyCycle);
 
```

Comment:
We want these reading to check the simulation and to compute gravity.

Reply:
We want to reduce things sent over Smartdashboard. This doesn't matter at comp. We can uncomment them when testing if needed.